src/store/constants/constants.js

```diff
@@ -8,3 +8,7 @@ export const APP_TOKEN_UPDATE = 'APP_TOKEN_UPDATE';
 
```

Comment:
Any reason to move `constants.js` to `constants/constants.js`? 🤔

Reply:
I'd rather have all constants in a single file for the moment. Thoughts?

Reply:
> 
> 
> Any reason to move `constants.js` to `constants/constants.js`? 🤔

i wanted to group all contstants in one folder.


> 
> 
> I'd rather have all action constants in a single file for the moment. Thoughts?

the moment all reducers scale you'll ask me destructure it, won't you? 👀

Reply:
No valid reason provided.